RecoTauTag/RecoTau/src/AntiElectronIDMVA6.cc

```diff
@@ -1269,3 +1325,30 @@ double AntiElectronIDMVA6::dCrackEta(double eta)
 
   return std::abs(retVal);
 }
+
+bool AntiElectronIDMVA6::atECalEntrance(const reco::Candidate* part, math::XYZPoint &pos)
+{
+  bool result = false;
+  double bField = 3.8; // hopefully Teslas are proper units (better to take it from event setup) 
+  BaseParticlePropagator theParticle =
+    BaseParticlePropagator(RawParticle(math::XYZTLorentzVector(part->px(),
+							       part->py(),
+							       part->pz(),
+							       part->energy()),
+				       math::XYZTLorentzVector(part->vertex().x(),
+							       part->vertex().y(),
+							       part->vertex().z(),
+							       0.)), 
+			   0.,0.,bField);
+  theParticle.setCharge(part->charge());
+  theParticle.propagateToEcalEntrance(false);
+  if(theParticle.getSuccess()!=0){
+    pos = math::XYZPoint(theParticle.vertex());
+    //mom = math::XYZTLorentzVector(theParticle.momentum());
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant